egress_backend/lambda/egress_api/update_request.py

```diff
@@ -81,13 +81,15 @@ def update_request(arguments: str, context: Any):
 
 # TO-DO: Inject Environment variables for reviewer group names
```

Comment:
I might have missed something but presumably the group names are stored somewhere configurable? If so, this todo doesn't make sense anymore?

Reply:
I'd assume not..... I've created an issue and will come back to it later
https://github.com/HicResearch/treehoose-egress-app-backend/issues/8